### PR TITLE
fix(components): [input] remove useless contants

### PR DESCRIPTION
**Pull request by pzzyf**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by pzzyf** [comment]
cc @chenxch  @kooriookami

File: packages/components/input/src/input.vue [hunk 1]
```diff
@@ -42,7 +42,7 @@
           :placeholder="placeholder"
           :style="inputStyle"
           :form="props.form"
```
Comment (chenxch) [inline packages/components/input/src/input.vue:44]: It should be possible to remove it here too.

Comment (pzzyf) [reply packages/components/input/src/input.vue:44]: image
can not remove

Comment (chenxch) [reply packages/components/input/src/input.vue:44]: Okay, I feel that this variable name should be returned to the control, and an alias should be given here, but it doesn't matter, so let's do this for now.

**Review by chenxch — APPROVED** [review]
Thank you for your contribution.

File: packages/components/input/src/input.vue (hunk 1 above)
Comment (kooriookami) [reply packages/components/input/src/input.vue:44]: > image can not remove

You can do like this:  const { form: elForm, formItem: elFormItem } = useFormItem(), there're some similar code you can find in other compoents.

**Comment by pzzyf** [comment]
@kooriookami

**Review by kooriookami — APPROVED** [review]

**Comment by kooriookami** [comment]
LGTM

**Review by btea — APPROVED** [review]

**Pull request merged**